cmd/execution_builder.go

```diff
@@ -838,7 +838,7 @@ func getContractEpochCounter(vm *fvm.VirtualMachine, vmCtx fvm.Context, view *de
 	// execute the script
 	err = vm.Run(vmCtx, script, view, p)
 	if err != nil {
-		return 0, fmt.Errorf("could not read epoch counter, script internal error: %w", script.Err)
```

Comment:
I'm sorry if this comes as being too picky, but are we now putting `script internal error` or just internal error? Its about the wording really